container/setup/setup-spark.sh

```diff
@@ -45,7 +45,7 @@ tar -xzvf ${SPARK_MEDIA}.tar.gz
 mv ${SPARK_MEDIA}/* $SPARK_HOME
 
 wget -q -O scopt_2.11-${SCOPT_211_VERSION}.jar \
-   http://central.maven.org/maven2/com/github/scopt/scopt_2.11/${SCOPT_211_VERSION}/scopt_2.11-${SCOPT_211_VERSION}.jar
+   https://repo1.maven.org/maven2/com/github/scopt/scopt_2.11/${SCOPT_211_VERSION}/scopt_2.11-${SCOPT_211_VERSION}.jar
```

Comment:
what happens if any of these wget fail ?

Reply:
bad things ... but atm we do this when we build the image once, and image build will fail when the above fails (though it's not elegant, it'll fail when we try to mv the file the line below.